service/admin.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+import (
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/model/user"
+)
+
+func (uis *UIServer) adminSettings(w http.ResponseWriter, r *http.Request) {
+	projCtx := MustHaveProjectContext(r)
+	data := struct {
+		ProjectData projectContext
+		User        *user.DBUser
+	}{projCtx, GetUser(r)}
```

Comment:
Check if GetUser returns nil.

Reply:
A nil user is handled later by the UI - it's what happens when you try to get to the page if not logged in

Reply:
sgtm